Hamins_Gas_Burners/A1.csv

```diff
@@ -1,8 +1,8 @@
-m,kW/m2,m,kW/m2
-r,Q(r),z,Q(z)
-0.081,0.0039,0.002,0.077826
-0.13,0,0.088,0.29217
-0.208,0,0.203,0.095652
-0.344,0,0.393,0.0052174
-0.6,0,0.668,0.0078261
-NaN,NaN,1.088,0.0030435
+m,kW/m2,m,kW/m2,kW
+r,Q(r),z,Q(z),Qb
+0.081,0.0039,0.002,0.077826,0.17
+0.13,0,0.088,0.29217,NaN
+0.208,0,0.203,0.095652,NaN
+0.344,0,0.393,0.0052174,Nan
```

Comment:
@ahaminsgithub @mcgratta Will "Nan" cause a problem?

Reply:
We have NaN in some of the verification exact csv files like water_evaporation_4.csv.

Reply:
Right, I was worried about the lower case "n" for that one particular instance "Nan".